osquery/tables/system/windows/system_util.cpp

```diff
@@ -89,36 +89,95 @@ std::string WmiResultItem::GetString(const std::string& name) const {
   return BSTR_to_string(value.bstrVal);
 }
 
+// Returns a pair with the first value indicating success of the query
+// and the second value being the data obtained from the query.
+std::pair<bool, bool> WmiResultItem::GetBool(const std::string& name) const {
```

Comment:
Are you saying return something like `-1 = query failed`, `0 = false`, `1 = true` and then just check the value on the callers end?